students/jstonham/README.md

```diff
@@ -2618,23 +2618,66 @@ Indices are generally stored in RAM, which is partly why they are so performant.
 - Redis - stored in RAM so does not require indices as it's already very performant.
 - ElasticSearch - useful for searching. As programmers generally use it to speed up searching, they tend to only store the search-related data here.
 
-## Creating Websites
+## Important Tools for Creating Websites
 
-React is a JavaScript library for building user interfaces. It takes the backend state and turns it into a frontend view.
+### React
 
-Gatsby JavaScript Framework is a tool based on React that builds websites using only static files, which increases the speed the user can navigate through the resulting interface.
+React is a JavaScript library or framework for building user interfaces.
+It takes the backend state and turns it into a frontend view.
+The framework takes care of the boilerplate so React allows programmers to simplify tasks that would otherwise be difficult.
+(In a similar way, JUnit is a Java framework for simplifying the writing of tests).
 
-Node.js provides the ability for programmers to write web servers using JavaScript. Node Package Manager (NPM) is a command line interface for managing node modules (a way of getting node applications such as VS Code and Atom).
+There is some debate regarding whether React should be thought of as a framework or a library.
+Due to React's popularity, many tools have been built around it, e.g. JSX (JavaScript Extended).
+React, in combination with these tools, is generally thought of as a framework.
+
+React is a powerful tool that allows classes, inheritance, nested components etc.
```

Comment:
I'm guessing your notes for this week will word slightly clearer so I vote we delete

```suggestion
```